README.md

```diff
@@ -1,89 +1,74 @@
+  * [Image object detection](https://github.com/jolibrain/deepdetect/tree/master/demo/objdetect) using python client
+  * [Image segmentation](https://github.com/jolibrain/deepdetect/tree/master/demo/segmentation) using python client
+* [Performance tools and report](https://github.com/jolibrain/dd_performances) done on NVidia Desktop and embedded GPUs, along with Raspberry Pi 3.
+* [References](#references)
+* [Authors](#authors)
+
+## Machine Learning functionalities per library
+
+|                   | Caffe | Caffe2 | XGBoost | TensorRT | NCNN | Libtorch | Tensorflow | T\-SNE | Dlib |
+|------------------:|:-----:|:------:|:-------:|:--------:|:----:|:--------:|:----------:|:------:|:----:|
+| **Stage**         |       |        |         |          |      |          |            |        |      |
+| Training \(CPU\)  | Y     | Y      | Y       | N/A      | N/A  | Y        | N          | Y      | N    |
+| Training \(GPU\)  | Y     | Y      | Y       | N/A      | N/A  | Y        | N          | Y      | N    |
+| Inference \(CPU\) | Y     | Y      | Y       | Y        | Y    | Y        | Y          | N/A    | Y    |
+| Inference \(GPU\) | Y     | Y      | Y       | Y        | Y    | Y        | Y          | N/A    | Y    |
+| **Task**          |       |        |         |          |      |          |            |        |      |
+| Classification    | Y     | Y      | Y       | Y        | Y    | Y        | Y          | N/A    | Y    |
+| Object Detection  | Y     | Y      | N       | Y        | Y    | N        | N          | N/A    | Y    |
+| Segmentation      | Y     | N      | N       | N        | N    | N        | N          | N/A    | N    |
+| Regression        | Y     | N      | Y       | N        | N    | N        | N          | N/A    | N    |
+| Autoencoder       | Y     | N      | N/A     | N        | N    | N        | N          | N/A    | N    |
+| OCR / Seq2Seq     | Y     | N      | N       | N        | Y    | N        | N          | N      | N    |
+| Time\-Series      | Y     | N      | N       | N        | Y    | N        | N          | N      | N    |
+| **Input Data**    |       |        |         |          |      |          |            |        |      |
+| CSV               | Y     | N      | Y       | N        |      | N        | N          | Y      | N    |
+| SVM               | Y     | N      | Y       | N        |      | N        | N          | N      | N    |
+| Text words        | Y     | N      | Y       | N        |      | N        | N          | N      | N    |
+| Text characters   | Y     | N      | N       | N        |      | N        | N          | Y      | N    |
+| Images            | Y     | Y      | N       | Y        |      | Y        | Y          | Y      | Y    |
+| Time\-Series      | Y     | N      | N       | N        |      | N        | N          | N      | N    |
+
+## Main features:
```

Comment:
Maybe we could restructure a bit the main features table ? Suggestions: serving / models / data.

Also, do we want main features to show ahead of ML features table ?

Reply:
done

Reply:
> Maybe we could restructure a bit the main features table ? Suggestions: serving / models / data.

Not sure I get it . You mean replacing `Stage / Task/ Input data` by `Serving / Models / Data` ?

> Also, do we want main features to show ahead of ML features table ?

I hesitate to do that, since we are two, I do it :)